stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -107,6 +76,19 @@ final class _ContiguousArrayStorage<Element> : _ContiguousArrayStorage1 {
   }
 
 #if _runtime(_ObjC)
+  /// If the `Element` is bridged verbatim, invoke `body` on an
+  /// `UnsafeBufferPointer` to the elements and return the result.
+  /// Otherwise, return `nil`.
+  internal final override func _withVerbatimBridgedUnsafeBuffer(
+    @noescape body: (UnsafeBufferPointer<AnyObject>) throws -> Element
+  ) rethrows -> Element? {
+    var result: Element? = nil
+    try self._withVerbatimBridgedUnsafeBufferImpl {
+      result = try body($0)
+    }
+    return result
+  }
+    
   /// If `Element` is bridged verbatim, invoke `body` on an
   /// `UnsafeBufferPointer` to the elements.
   internal final override func _withVerbatimBridgedUnsafeBufferImpl(
```

Comment:
Does this function still override anything?  Do we need to factor it as a separate function, or can we inline it into its only (as far as I can see) caller -- `_withVerbatimBridgedUnsafeBuffer `?